psh/tools/psh.py

```diff
@@ -119,6 +124,68 @@ def assert_cmd_successed(pexpect_proc):
     assert _get_exit_code(pexpect_proc) == 0, 'The exit status of last passed command does not equal 0!'
 
 
+def ls(pexpect_proc, dir=''):
+    ''' Returns the list with named tuples containing information about files present in the specified directory '''
+    File = namedtuple('File', ['name', 'owner', 'is_dir', 'date'])
+    Date = namedtuple('Date', ['month', 'mday', 'time'])
```

Comment:
It seems to be not supported, because they are displayed the same way. I've created issue for that (https://github.com/phoenix-rtos/phoenix-rtos-project/issues/552), but would rather leave it here for now.